js/character.js

```diff
@@ -0,0 +1,29 @@
+import { colRef, getDocs } from '../firebase.js';
+
+getDocs(colRef)
+  .then((snapshot) => {
+    snapshot.docs.forEach((doc) => {
```

Comment:
snapshot 이라는 변수가 이 then 콜백에서 안 쓰이자나요?? 그럴 땐 가져올 때부터 비구조화 할당으로 가져와서 사용하는 것도 방법입니다.
```suggestion
  .then(({ docs }) => {
    docs.forEach((doc) => {
```

Reply:
헉 알려주셔서 정말 감사합니다!! 비구조화 할당에 대해서도 더 자세히 공부해보도록 하겠습니다..!